src/controllers/user.controller.js

```diff
@@ -0,0 +1,95 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
```

Comment:
There is a logical error here. The `res.send(users);` line should not be executed if `users` is not found. You should only send a 404 status in this case.